tasks/update.yml

```diff
@@ -13,12 +13,22 @@
     list_files: yes
```

Comment:
This task extracts the archive to `/tmp/dokuwiki`, not to `/srv/dokuwiki` like in the task title.